### ISSUE TITLE
[RTE] Incorrect pasting of copied links

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#542]
## Description
Text editor incorrectly displays the words to which the hyperlink belongs.

#### Steps to Reproduce

1. Open this word document [AXS module_for bug report.docx](https://github.com/epam/UUI/files/8700230/AXS.module_for.bug.report.docx)
2. Copy all text from the document.
3. Open [Rich Text Editor](https://uui.epam.com/documents?category=components&id=richTextEditor&mode=doc&skin=UUI4_promo)
4. Click on the RTE area.
5. Paste copied text.
6. Pay attention on the last paragraph.

#### Actual result
Part of the link goes to other words.

https://user-images.githubusercontent.com/57751283/168601432-406d7516-df26-4785-8ef5-0f84f350bf72.mp4

#### Expected result
Link in Word and RTE looks the same.

#### Steps to Reproduce

1. Open this word document [IHS Markit module_for bug report.docx](https://github.com/epam/UUI/files/8703488/IHS.Markit.module_for.bug.report.docx)
2. Copy all text from the document.
3. Open [Rich Text Editor](https://uui.epam.com/documents?category=components&id=richTextEditor&mode=doc&skin=UUI4_promo)
4. Click on the RTE area.
5. Paste copied text.
6. Pay attention on the last paragraph.

#### Actual result
Part of the link goes to other words.

https://user-images.githubusercontent.com/57751283/168684476-50813d62-52be-4f5b-b03d-286eb3e2f647.mp4

#### Expected result
Link in Word and RTE looks the same.

#### Environment
* UUI version: 4.6.3
* Browser name: Chrome 101.0.4951.54
* Operating System: Win 10

## Description
Numbered links copied from "Notes" are pasted incorrectly

#### Steps to Reproduce

1. Create a new Note.
2. Paste the following links into your note:
2.1. https://www.atlassian.com/agile/scrum/scrum-metrics#:-text=What9620are%20scrum%20metrics963F.improve%20a%20team's%20agile%20journey
2.2. https://github.com/epam/UUI
3. Mark them out with 1) and 2)
4. Copy the links together with the numbers
5. Paste them into the RTE.

#### Actual result
URL doesn't understand as a link

https://user-images.githubusercontent.com/57751283/228965611-b0816e74-3c3e-4c28-bf4e-ea1a4794e20b.mp4

#### Expected result
Links detected.

#### Note
Most likely, this is because the links are preceded by numbers in parentheses.

#### Environment
* UUI version: 4.10.2
* Browser name: any
* Operating System: any

**Comment by NatalliaAlieva** [comment]
After a closer investigation of Сase #3, we have concluded that when copying links from the Notes app, the links do not necessarily need to be inserted as hyperlinks. Below is attached a video that shows the copy/paste link behavior in the Google Doc. Therefore in the new version of RTE we have made link insertion as plain text.

https://github.com/epam/UUI/assets/57751283/db5d2d49-1705-4d4a-8142-15d0bcb8711c
